app/views/provider_interface/decisions/respond.html.erb

```diff
@@ -20,8 +20,11 @@
       <%- offer_text = 'Make an offer' %>
       <%- reject_text = 'Reject application' %>
       <%= f.govuk_radio_buttons_fieldset :decision, legend: { size: 'm', text: 'Choose response' } do %>
-        <%= f.govuk_radio_button :decision, 'offer', label: { text: offer_text }, link_errors: true %>
-        <%= f.govuk_radio_button :decision, 'reject', label: { text: reject_text } %>
+        <%= f.govuk_radio_button :decision, 'new_offer', label: { text: offer_text }, link_errors: true %>
+        <%= f.govuk_radio_button :decision, 'edit_course', label: { text: "#{offer_text} but change course" }, link_errors: true %>
```

Comment:
Please hide all change offer options unless `provider_change_response` feature flag is active.

Reply:
@mnacos :+1: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1667/commits/153dc4315f16201e6608867b2f83e42de58bc845